### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by jakub-bartoszek**
I fixed a few responsive and logic troubles. Check it out 😸

### CODE REVIEW

**Comment by AndriiStafiniak** [comment]
Wow, it looks great, much more professional now 🚀 Just one small thing to improve. I know it wasn't your task, but since you handle corrections, it would be fantastic if you could address it 😃We need to keep at least 2 columns on 320px wide screens. The problem is in the screenshots below
Zrzut ekranu 2023-11-24 o 23 57 06
Zrzut ekranu 2023-11-24 o 23 56 20
Zrzut ekranu 2023-11-24 o 23 55 44
Zrzut ekranu 2023-11-24 o 23 54 39

File: src/components/common/Tile/PersonTile/styled.jsx [hunk 1]
```diff
@@ -1,83 +1,59 @@
...
-    min-height: 169px;
-  }
+export const ImageWrapper = styled.div`
+	border-radius: 5px;
+	background-color: ${({ theme }) => theme.colors.tile.imageWrapper};
+	aspect-ratio: 6/9;
+	display: flex;
+	justify-content: center;
+	align-items: center;
 `;
```
Comment (AndriiStafiniak) [inline src/components/common/Tile/PersonTile/styled.jsx]: I think this change is causing problems
Zrzut ekranu 2023-11-25 o 02 17 09

**Review by AndriiStafiniak — CHANGES_REQUESTED** [review]

File: src/components/common/Tile/Tile.jsx [hunk 2]
```diff
@@ -1,59 +1,73 @@
...
+					src={`https://image.tmdb.org/t/p/w500/${poster_path}`}
+					alt=""
+				/>
+			</ImageWrapper>
+			<Content>
+				<Title>{title}</Title>
+				<Year>{year}</Year>
+				<Release>
+					<Production>{countryNames.toString().replace(",", ", ")}</Production>
+					<ReleaseDate>{release_date.toString().replaceAll("-", ".")}</ReleaseDate>
```
Comment (AndriiStafiniak) [inline src/components/common/Tile/Tile.jsx]: It's great that you noticed it, but we also need to change the format. For example using release_date.toString().split("-").reverse().join(".")  I'll fix it, it's my mistake😃
Zrzut ekranu 2023-11-25 o 02 42 06
Zrzut ekranu 2023-11-25 o 02 42 19

**Comment by jakub-bartoszek** [comment]
Thanks for feedback, i will fix that!

**Comment by jakub-bartoszek** [comment]
I think it's working now!

**Comment by szymonbasiul** [comment]
Good changes, its ok with me.

**Review by szymonbasiul — APPROVED** [review]
Its ok with me, good job ;)

**Comment by jakub-bartoszek** [comment]
I think i will change a few more things!

**Pull request closed without merging**